core/src/main/java/com/yahoo/imapnio/async/client/ImapAsyncSession.java

```diff
@@ -13,6 +13,15 @@
  * A class that defines the behavior of Asynchronous IMAP session.
  */
 public interface ImapAsyncSession {
+    /**
```

Comment:
can we turn on logging if session is already established?

Reply:
Done. Added setDebugMode() in ImapSession.